### PR TITLE
CDAP-7182 Update docs for cdap script

### PR DESCRIPTION
**Pull request by wolf31o2**
This is currently a work-in-progress. I've gotten the low-hanging fruit.

Completed:
- [x] `svc-master run` -> `cdap run`
- [x] `cdap-cli.sh` -> `cdap cli`
- [x] `cdap-cli.bat` -> `cdap.bat cli`
- [x] `cdap.sh` -> `cdap sdk`
- [x] `cdap.bat` -> `cdap.bat sdk`
- [x] `tx-debugger.sh` -> `cdap tx-debugger`

This requires that caskdata/cdap#6774 is merged, first.

### CODE REVIEW

**Comment by awholegunch** [comment]
Running as a Quick Build: http://builds.cask.co/browse/CDAP-DQB138-1

File: cdap-docs/developers-manual/source/getting-started/building-apps.rst [hunk 1]
```diff
@@ -41,8 +41,8 @@ to use the namespace *my_namespace*, you would replace ``default`` with ``my_nam
 Accessing CLI, curl, and the SDK bin
 ------------------------------------ 
 
-- For brevity in the commands given below, we will simply use ``cdap-cli.sh`` for the CDAP
-  Command Line Interface. Substitute the actual path of ``./<CDAP-SDK-HOME>/bin/cdap-cli.sh``,
+- For brevity in the commands given below, we will simply use ``cdap cli`` for the CDAP
+  Command Line Interface. Substitute the actual path of ``./<CDAP-SDK-HOME>/bin/cdap cli``,
   or ``<CDAP-SDK-HOME>\bin\cdap-cli.bat`` on Windows, as appropriate.
```
Comment (awholegunch) [inline cdap-docs/developers-manual/source/getting-started/building-apps.rst]:
```
or ``<CDAP-SDK-HOME>\bin\cdap cli`` on Windows, as appropriate.

Comment (awholegunch) [inline cdap-docs/developers-manual/source/getting-started/building-apps.rst]: Probably need to search and replace on `cdap-cli.bat`.

File: cdap-docs/developers-manual/source/getting-started/quick-start.rst [hunk 2]
```diff
@@ -42,12 +42,12 @@ In the examples and commands that follow, for brevity we will use these conventi
   on a UNIX-type system or Windows.
   
 - The `CDAP Command Line Interface (CLI) <http://docs.cask.co/cdap/current/en/reference-manual/cli-api.html>`__
-  is included in the SDK in the ``bin`` directory, either at ``bin/cdap-cli.sh`` or |---|
+  is included in the SDK in the ``bin`` directory, either at ``bin/cdap cli`` or |---|
   on Windows |---| ``bin\cdap-cli.bat``. The CLI allows you to quickly access CDAP
```
Comment (awholegunch) [inline cdap-docs/developers-manual/source/getting-started/quick-start.rst]:
```
``bin\cdap cli``

File: cdap-docs/developers-manual/source/getting-started/quick-start.rst [hunk 3]
```diff
@@ -42,12 +42,12 @@ In the examples and commands that follow, for brevity we will use these conventi
...
   facilities from a command line environment.
   
-- For brevity in the commands given below, we will simply use ``cdap-cli.sh`` for the CDAP
-  Command Line Interface. Substitute the actual path of ``./<CDAP-SDK-HOME>/bin/cdap-cli.sh``,
+- For brevity in the commands given below, we will simply use ``cdap cli`` for the CDAP
+  Command Line Interface. Substitute the actual path of ``./<CDAP-SDK-HOME>/bin/cdap cli``,
   or ``<CDAP-SDK-HOME>\bin\cdap-cli.bat`` on Windows, as appropriate.
```
Comment (awholegunch) [inline cdap-docs/developers-manual/source/getting-started/quick-start.rst]:
```
``<CDAP-SDK-HOME>\bin\cdap cli``

File: cdap-docs/developers-manual/source/getting-started/building-apps.rst (hunk 1 above)
Comment (wolf31o2) [inline cdap-docs/developers-manual/source/getting-started/building-apps.rst]: No. Windows hasn't been updated. There's a separate JIRA for that.

**Comment by awholegunch** [comment]
Re-running as Quick Build: http://builds.cask.co/browse/CDAP-DQB138-2 (killed as I noticed errors.)

**Comment by awholegunch** [comment]
Passes as Quick Build: http://builds.cask.co/browse/CDAP-DQB138-3

**Comment by awholegunch** [comment]
Doc changes LGTM.

**Pull request merged**